Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -643,15 +663,15 @@ eval BackPressed state = do
     FindingEstimate -> do
                       _ <- pure $ updateLocalStage SearchLocationModel
                       continue state{props{rideRequestFlow = false, currentStage = SearchLocationModel, searchId = "", isSource = Just false,isSearchLocation = SearchLocation}}
-    QuoteList       -> continue state{ props{isPopUp = if ( not null (filter (\a -> a.seconds > 0) state.data.quoteListModelState)) then ActiveQuotePopUp else ConfirmBack}}
+    QuoteList       -> if state.props.customerTip.enableTips then continue state{props {customerTip{isTipSelected = true, tipForDriver = 10, tipActiveIndex=1}, isPopUp = TipsPopUp} }else continue state{ props{isPopUp = if ( not null (filter (\a -> a.seconds > 0) state.data.quoteListModelState)) then ActiveQuotePopUp else ConfirmBack}}
```

Comment:
`state{props {customerTip{isTipSelected = true, tipForDriver = 10, tipActiveIndex=1}}` as this state we are using at multiple places
Can we create a function which will return this state and then use it